lib/renderers/server.js

```diff
@@ -9,14 +9,17 @@ import App from 'components/App';
 const serverRender = async () => {
 	
 	const dataURL = `http://${config.host}:${config.port}/data`;
-	const response = await axios.get(dataURL);		
-	const store = new StateApi(response.data);		
-	
-	return {
-			initialMarkup : ReactDOMServer.renderToString(
-		<App store = {store} />
-	), initialData : response.data 
-	};		
+	const response = await axios.get(dataURL);
+	const dateString = new Date().toString();
+	const store = new StateApi(response.data, dateString) ;		
+
+	return  {
+				initialMarkup : 
+					ReactDOMServer.renderToString(
+						<App store = {store} />
+					),
+				initialData :  [response.data, dateString] 

```

Comment:
What about passing store and so that same store can be used in both renderers is there any way to do that in react from Server to Client renderer

Reply:
That could make life super easy but need to do some research on that.